### PR TITLE
Rocket-Mission PR

### PR DESCRIPTION
**Pull request by Javed-Shahbaz**
- Use React documentation
- Use React components
- Use React props
- Use React Router
- Connect React and Redux
- Handle events in a React app
- Write unit tests with React Testing Library
- Use styles in a React app
- Use React hooks
- Apply React best practices and language style guides in code
- Use store, actions and reducers in React
- Perform a code review for a team member

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @Javed-Shahbaz @emarkees 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesnt have a professional readme. kindly consider addint the readme from microverse redme template and fill it according to your project requirements and pay special attention to the requiements while filling the relevant information in readme.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Profile.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Profile = () => (
+  <>
+    <div>
+      <h2>Our Profile</h2>
+    </div>
+  </>
+);
+
```
Comment (ichala) [inline src/components/Profile.js:10]: - as part of the requirements The My Profile section should display all reserved rockets and space missions. currently its empty so kindly implement that requirements 🙏

File: src/App.js [hunk 2]
```diff
@@ -12,7 +14,8 @@ function App() {
         <NavLink />
         <Routes>
           <Route path="/rockets" element={<Rockets />} />
-
+          <Route path="/Missions" element={<Missions />} />
```
Comment (ichala) [inline src/App.js:17]: - [ ] As part of the requirements you should add unit & integration tests for  for all your components and functions. kindly add that

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Team
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Good design ✔️
- Data fetched from API ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello Team! 👋 

### [OPTIONAL]
- In the future, Please write descriptive commit messages that are easy to understand and look professional. Kindly try to write commit messages that are relevant to actions that you have performed in that commit. 👍 

Screenshot 2023-06-04 at 2 58 19 AM

### [OPTIONAL]
Please DO NOT disable eslint linter, as they're there to make you write professional code. Please remove all comments where you've disabled eslint linter. 👍 

Screenshot 2023-06-04 at 3 01 37 AM


### Good Job for writing Test Cases asked by the previous Code Reviewer. Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**